### PR TITLE
1-day Sessions

### PR DESCRIPTION
**Pull request by ric9176**
This PR is to add the one day sessions waiting list to the site and a new form to handle that. There is still some small work to be done on styling this (the form checkboxes) and a bit of refactor and tidy up, but I'd like to get it live asap so people can get on the wait list. The important thing to review asap is the text as it was written very fast.

### CODE REVIEW

File: src/pages/thanks-for-signing-up.js [hunk 1]
```diff
@@ -6,49 +6,45 @@ import Header from '../components/layout/Header'
...
-        </TopSection>
-      </React.Fragment>
-    )
-  }
-}
+const ThanksForSignUp = () => (
+  <React.Fragment>
+    <Header
+      titleLines={['Thanks - your free resources are on their way!']}
+      subtitle="Every week for the next 5 weeks, you'll get a new email with juicy React goodness... You don't have to do a thing."
```
[the remark this answers is not in the document]
Comment (ric9176) [reply src/pages/thanks-for-signing-up.js]: They wouldn’t receive the emails again bc auto pilot stops you sending the same email twice to any contact. Not that it changes the result but I also imagine if they have already got the recourses they wouldn’t check the box asking for them again either.

File: src/pages/react-workshops.js [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+import { H2, P, H4, H5 } from '../components/text'
+import {
+  UpcomingTrainingSection,
+  withUpcomingTrainings,
+} from '../components/training'
+import Header from '../components/layout/Header'
+import { Card, Newsletter } from '../components/elements'
+import { GREY2 } from '../config/styles'
+
+const WorkshopCard = styled.div`
```
[the remark this answers is not in the document]
Comment (ric9176) [reply src/pages/react-workshops.js]: I agree ☝️Just to note, the reason this happened is bc we did this in parallel very quick on Friday (initial build of the 2 pages) and haven’t yet done the final pairing to clean up 🧹 this branch. As I mentioned it was premature for PR but I wanted to get it in early bc it’s important we agree about the text (names and descriptions) ASAP to get it live. I’ve been committing plenty this weekend but didn’t yet get to tidy up this page fully as I’ve been focused on getting the form integration working. I think you’re right we should use  Storybook. Further to this I’ve been noticing there are many redactora and cleanups we need to do as there is a bit of tech debt building

File: src/pages/interest-form.js [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+    email,
+    fundamentals = false,
+    styling = false,
+    hooks = false,
+    perf = false,
+    gqlclient = false,
+    testing = false,
+    resources = false,
+  }) => {
+    console.log(fundamentals)
```
Comment (alexlbr) [inline src/pages/interest-form.js]: @ric9176  do we need this console.log?

File: src/components/form/FieldInput.js [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import React, { Fragment } from 'react'
+import React from 'react'
```
Comment (alexlbr) [inline src/components/form/FieldInput.js]: @ric9176  can we rename FieldInput to InputFiled

File: src/components/form/Checkbox.js [hunk 5]
```diff
@@ -0,0 +1,75 @@
...
+    <HiddenCheckbox checked={checked} {...props} />
+    <StyledCheckbox checked={checked} label={label} {...props}>
+      <Icon viewBox="0 0 24 24">
+        <polyline points="20 6 9 17 4 12" />
+      </Icon>
+    </StyledCheckbox>
+  </React.Fragment>
+)
+
+const CheckBoxContainer = props => {
```
Comment (alexlbr) [inline src/components/form/Checkbox.js]: @ric9176 can we rename the CheckBoxContainer to CheckboxField and the file to CheckboxField.js please

**Pull request merged**